src/Symfony/ResponseAssertions.php

```diff
@@ -20,4 +20,47 @@ protected static function assertJsonResponse(array $expected, JsonResponse $resp
         $content = json_decode($response->getContent(), true);
         Assert::assertSame($expected, $content);
     }
+
+    protected static function assertResponse(Response $response, int $expectedStatusCode, ?string $expectedMessage = null): void
```

Comment:
Use expectedContent if you're comparing with response->getContent (also for other methods)